editor/editor_node.h

```diff
@@ -263,6 +265,7 @@ class EditorNode : public Node {
 	EditorResourcePreview *resource_preview = nullptr;
 	EditorSelection *editor_selection = nullptr;
 	EditorSettingsDialog *editor_settings_dialog = nullptr;
+	FindInFilesDialog *find_in_files_dialog = nullptr;
```

Comment:
Maybe changes in header and constructor (`find_in_files_dialog` assignment) could be moved directly to `case PROJECT_FIND_IN_FILES: {` (to keep variable local).